os/net/mac/tsch/tsch-slot-operation.c

```diff
@@ -370,6 +370,18 @@ get_packet_and_neighbor_for_link(struct tsch_link *link, struct tsch_neighbor **
 
   return p;
 }
+
+static
+void tsch_slot_operation_update_current_bacokff(void){
+    if(current_link != NULL
+        && (current_link->link_options & LINK_OPTION_TX)
+        && (current_link->link_options & LINK_OPTION_SHARED) )
```

Comment:
This line should look like this: no space between `) )` and `{` in the same line:

         && (current_link->link_options & LINK_OPTION_SHARED)) {